### PR TITLE
Create an Azure Service Bus namespace with one topic and subscription

### PR DESCRIPTION
**Pull request by lgonczar**
Create an Azure Service Bus namespace with one topic and subscription.

Here are the related pull requests creating modules used from here:
https://github.com/hmcts/terraform-module-servicebus-namespace/pull/1
https://github.com/hmcts/terraform-module-servicebus-topic/pull/1
https://github.com/hmcts/terraform-module-servicebus-subscription/pull/1

### CODE REVIEW

**Comment by rkondratowicz** [comment]
should this not be a part of some existing project repo? (IDAM?)

**Comment by lgonczar** [comment]
As far as I know, that solution is not ready yet.

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  source          = "git@github.com:hmcts/terraform-module-servicebus-namespace.git"
+  name            = "${local.namespace_name}"
+  location        = "${var.location}"
+  resource_group_name  = "${local.resource_group_name}"
+}
+
+module "private_beta_activation_topic" {
+  // TODO: reference a specific release
+  source          = "git@github.com:hmcts/terraform-module-servicebus-topic.git"
+  name            = "reform.private-beta.account.activation"
```
Comment (rkondratowicz) [inline main.tf]: shouldn't this name have 'idam' somewhere?

Comment (lgonczar) [reply main.tf]: this topic is in `servicebus-idam` namespace

**Review by rkondratowicz — APPROVED** [review]

**Pull request merged**